spec/features/basic_search_spec.rb

```diff
@@ -4,11 +4,19 @@
 
 RSpec.feature "Verify query search results" do
 
-  subject { Page.new }
+  # subject { Page.new }
+  # subject { Home.new }
 
   background do
-    subject.visit 'https://google.com'
-    subject.enter_keyword keyword
+    # subject.visit 'https://google.com'
+    # subject.enter_keyword keyword
+  @page = Home.new
+    @page.load
```

Comment:
Done.
Not sure if I understood you correctly though. Coz now it doesn't make difference besides more descriptive method name, imo.